controllers/Alerts.js

```diff
@@ -67,7 +68,8 @@ AlertRouter.put('/edit', function(req, res) {
     alert.set({
       name: alertToEdit.name,
       description: alertToEdit.description,
-      isSmsSent: alertToEdit.isSmsSent
+      isSmsSent: alertToEdit.isSmsSent,
+      sentDate: alertToEdit.sentDate
```

Comment:
Would suggest `timestamp` or `sentTime` rather than `sentDate` but it's just wording.  Will approve anyway!